### ISSUE TITLE
GDB hangs if you start the stack view before the debugger

### ISSUE THREAD

**Issue by snare** [snare/voltron#105]
If the stack or memory view is started before the debugger has been launched and already hit a breakpoint, GDB will hang when you first step. It seems to be that GDB hangs when `x/s` is called on certain addresses (even without Voltron loaded). Needs further investigation.

**Comment by dianapinkberry** [comment]
Please, can you help me. Sorry for my bad English. I tried to install voltron.What my problem is?I'm new,sorry if I write not in that topic.
First Terminal
```
gdb -q first
Voltron loaded.
Reading symbols from first...(no debugging symbols found)...done.
(gdb) b main
Breakpoint 1 at 0x100000f24
```

Second Terminal
```
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/voltron-0.1-py2.7.egg/voltron/view.py", line 252, in run
    msg = e.message.args[1].strerror
IndexError: tuple index out of range

Error: HTTPConnectionPool(host='localhost', port=5555): Max retries exceeded wi>
```

**Comment by snare** [comment]
Per the readme, the Voltron server isn't started until the program you're debugging ("first" in this example) is running. Run the inferior and the view should be able to connect.

```(gdb) run```

**Comment by dianapinkberry** [comment]
New problem:( Sorry if I misunderstand something here 
First Term
```
$ sudo gdb -q ./a.out
Voltron loaded.
Reading symbols from ./a.out...
warning: dsym file UUID doesn't match the one in /Users/a1/a.out
(no debugging symbols found)...done.
(gdb) voltron init
(gdb) set disassembly-flavor intel
(gdb) b main
Breakpoint 1 at 0x100000f24
(gdb) run
Starting program: /Users/a1/a.out 

Breakpoint 1, 0x0000000100000f24 in main ()
(gdb) 
```
Second Term(voltron view stack)

```
Encountered an exception while running the view 'StackView':
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/voltron-0.1-py2.7.egg/voltron/main.py", line 53, in main
    inst.run()
  File "/Library/Python/2.7/site-packages/voltron-0.1-py2.7.egg/voltron/view.py", line 239, in run
    self.render()
  File "/Library/Python/2.7/site-packages/voltron-0.1-py2.7.egg/voltron/plugins/view/memory.py", line 145, in render
    super(StackView, self).render()
  File "/Library/Python/2.7/site-packages/voltron-0.1-py2.7.egg/voltron/plugins/view/memory.py", line 53, in render
    api_request('targets', block=self.block))
  File "/Library/Python/2.7/site-packages/voltron-0.1-py2.7.egg/voltron/core.py", line 388, in send_requests
    t.join()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 949, in join
    self.__block.wait()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 340, in wait
    waiter.acquire()
  File "/Library/Python/2.7/site-packages/voltron-0.1-py2.7.egg/voltron/view.py", line 329, in sigwinch_handler
    self.do_render()
  File "/Library/Python/2.7/site-packages/voltron-0.1-py2.7.egg/voltron/view.py", line 309, in do_render
    self.ft_body = self.body
AttributeError: 'StackView' object has no attribute 'body'
```

voltron view register is empty

**Comment by snare** [comment]
Can you please enable debug logging by putting the following into `~/.voltron/config`:

```
{"general": {"debug_logging": true}}
```

Then repeat the steps to reproduce the issue, and post your `~/.voltron/voltron_*.log` files.

Also open a new issue, this is not relevant to this issue.

**Comment by cx9527** [comment]
Did you found a solution yet? I got the same problem, i just delete  `voltron view stack` simply  in tmuxinator config file. So i have to launch tmuxinator first,and then open the stack view when the debuggee launched.

**Comment by snare** [comment]
Not yet. I will investigate.

**Comment by snare** [comment]
Fixed in 430cc8a51befa186769bb6514c4089d717f74a19
